### PR TITLE
Fix error adding or deleting new fields

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description

Fixed problem with new or missing columns in agent table.
 
### Issues Resolved

- #4562

### Evidence

image

image

### Test
The tests include manual tests

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by jbiset — DISMISSED** [review]
## Resume

Code Review - :green_circle: 

Test - :green_circle: 

## Evidence

<details>

Capture_Evidence

RunTest_Evidence

</details>

**Review by chantal-kelm — APPROVED** [review]
CR: ✅ 
Test: ✅ 

Captura de pantalla 2023-07-06 a la(s) 15 18 06

**Comment by chantal-kelm** [comment]
Changelog is missing

**Review by chantal-kelm — APPROVED** [review]
Captura de pantalla 2023-07-20 a la(s) 12 39 44

File: plugins/main/public/controllers/agent/components/agents-table.js [hunk 1]
```diff
@@ -252,15 +252,14 @@ export const AgentsTable = withErrorBoundary(
         const rawAgents = await this.props.wzReq('GET', '/agents', {
           params: { ...this.buildFilter(), select: selectFields },
         });
-        const formatedAgents = (
-          ((rawAgents || {}).data || {}).data || {}
-        ).affected_items.map(this.formatAgent.bind(this));
+        const formatedAgents = rawAgents?.data?.data?.affected_items.map(
```
Comment (jbiset) [inline plugins/main/public/controllers/agent/components/agents-table.js:255]: Question:
In this line if rawAgents?.data?.data?.affected_items is undefined, then undefined.map throws an error. Right?

**Review by jbiset — CHANGES_REQUESTED** [review]
CR: :white_check_mark: 
Test : :heavy_check_mark: 

## Test evidence

image

File: plugins/main/public/controllers/agent/components/agents-table.js (hunk 1 above)
Comment (yenienserrano) [reply plugins/main/public/controllers/agent/components/agents-table.js]: Done, thanks for letting me know!

**Review by jbiset — APPROVED** [review]
LGTM

Code Review - :green_circle: 

Test - :green_circle: 

## Evidence
### UI
image

### Test
image

**Review by chantal-kelm — APPROVED** [review]

**Pull request merged**